dev/sbt-plugin/src/sbt-test/sbt-plugin/run-services/test

```diff
@@ -0,0 +1,77 @@
+# Structure of this test:
+# =======================
+
+# We test that the service locator can be started and stopped on demand, and that services are automatically 
+# registered to the service locator.
+
+# Service locator tests
+# --------------
+
+# Precondition
+> validateRequest http://localhost:8000 should-be-down
+
+> lagomServiceLocatorStart
+> validateRequest retry-until-success http://localhost:8000
+> validateRequest http://localhost:8000/services/fooservice status 404
+
+# Check that the external service is started
+> validateRequest http://localhost:8000/services/externalservice status 200
+
+# Run the service and check that it eventually gets registered
+> run
+> validateRequest retry-until-success http://localhost:8000/services/fooservice status 200
+
+# Stop the service locator and check that it does stop
+> lagomServiceLocatorStop
+> validateRequest retry-until-success http://localhost:8000 should-be-down
+
+# Stop the service
+> stop
+
+# Disabled service locator and cassandra tests
```

Comment:
What do you mean with "and cassandra tests"? There is no cassandra test in this section as far as I can tell